test/ringpop-assert.js

```diff
@@ -224,16 +233,11 @@ function changeStatus(t, tc, sourceIx, subjectIx, status, subjectIncNoDelta) {
     return f;
 }
 
-/**
- * Send a ping to the SUT from a fake node
- *
- * @param {Test} t The current running test
- * @param {TestCoordinator} tc The test coordinator.
- * @param {number} nodeIx The index of the fakeNode that should send the ping.
- * @param {object} piggybackOpts The changes to piggy back on the ping.
- * @param {object} bodyOverrides Overwrite specific ping body parameters (checksum, source, sourceIncarnationNumber, changes)
- * @returns {Function} Returns the function that'll be invoked when running the integration test.
- */
+
+// Send a ping to the SUT from a fake node, where:
+// - nodeIx is the index of the fakeNode that should send the ping.
```

Comment:
The deleting of the comment format is typically not a cool thing to do (unless it was decided that we're not going that route).